integration/js-routes-test.ts

```diff
@@ -0,0 +1,42 @@
+import { test } from "@playwright/test";
```

Comment:
I went to add a unit test to our build in `remix-dev` but it looks like that test is completely skipped and relies on the old Gists app that no longer exists. I tried to adapt it with a new fixture but couldn't get it working, so I went with this for now. We should probably fix that test file though.